src/createApp.js

```diff
@@ -4,19 +4,33 @@ const fs = require('fs');
 const os = require('os');
 const ncp = require('ncp').ncp;
 const { exec } = require('child_process');
+const chalk = require('chalk');
```

Comment:
Delete this reference(it wasn't included on the package.json file anyway), we will get:
`
Error: Cannot find module 'chalk'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at Object.<anonymous> (/home/snow/Documents/create-proton-app/src/createApp.js:7:15)
    at Module._compile (internal/modules/cjs/loader.js:701:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:712:10)
    at Module.load (internal/modules/cjs/loader.js:600:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:539:12)
    at Function.Module._load (internal/modules/cjs/loader.js:531:3)
`